test/helpers/xcm.ts

```diff
@@ -181,6 +181,28 @@ export async function injectHrmpMessageAndSeal(
   return block;
 }
 
+function patchLocation(value: any) {
```

Comment:
This `patchLocation` method is specific for XCM versions above v3. Now that it is no longer used inside `as_v4`, we should add a comment for easier maintenance.